### ISSUE TITLE
 Need to set the default cli command

### ISSUE THREAD

**Issue by liyikun** [microsoft/vscode-react-native#1095]
## Actual Behavior
1.  The default command used is react-native, but we use is crn-cli

## Expected Behavior
1. Support setting normal cli ,  normal is 'react-native run-ios' , I would like to be able to run the command '[default-cli] run-android ' when starting the debugger from Visual Studio Code.
2. Example 'crn-cli run-ios'

## Software versions
- React Native version: 
- https://github.com/ctripcorp/CRN

**Comment by liyikun** [comment]
[https://github.com/microsoft/vscode-react-native/blob/229e4d5170d2ce55c5c34430e3b5df4af82d4813/src/common/commandExecutor.ts](url)

```
    private static ReactNativeCommand = "react-native";
    private static ReactNativeVersionCommand = "--version";
```

Support  diy

**Comment by SounD120** [comment]
Hey guys. I am closing this because this enhancement will be implemented as part of #1101.
